pkg/utils/sysfs.go

```diff
@@ -63,6 +67,17 @@ func SetCPUScalingMaxFreq(cpu ID, freq int) error {
 	return setCPUFreqValue(cpu, "scaling_max_freq", freq)
 }
 
+// SetCPUsScalingGovernor sets the scaling_governor of a given set of CPUs
+func SetCPUsScalingGovernor(cpus []ID, governor string) error {
```

Comment:
nit: I'm sorry if I am an unbearable pain in the arse, and I suspect I am since I did a bad first review round and did not comment on this before. And therefore please feel free to ignore this, it's a mere 'hey we could do this, too' suggestion... I'd call this functions `SetScalingGovernorForCPUs()` because that's more readable and less easy to mix up with `SetCPUScalingGovernor()`.

Reply:
Thanks, fixed